src/test/java/net/snowflake/client/jdbc/MultiStatementLatestIT.java

```diff
@@ -222,12 +222,12 @@ public void testMultiStmtExecuteQuery() throws SQLException {
     assertFalse(rs.next());
 
     // second statement
-    assertFalse(statement.getMoreResults());
```

Comment:
We also had a small bug where getMoreResults() was returning false when it should have been returning true after a call to executeQuery(). The bug did not happen for after calls to execute() or executeUpdate(). This is because the logic depends on the parent query ID being non-null and query IDs were not getting properly populated for executeQuery() calls (but they were for execute() and executeUpdate() calls)